src/coreclr/inc/readytorun.h

```diff
@@ -397,8 +397,6 @@ enum ReadyToRunHelper
     // Floating point ops
     READYTORUN_HELPER_DblRem                    = 0xE0,
     READYTORUN_HELPER_FltRem                    = 0xE1,
-    READYTORUN_HELPER_DblRound                  = 0xE2,
```

Comment:
These needs to stay. It would require `MINIMUM_READYTORUN_MAJOR_VERSION` bump to get rid of them. You can add a comment that these can be deleted once `MINIMUM_READYTORUN_MAJOR_VERSION` is 10 or higher.

Reply:
I was afraid of that. I'll revert that part.